src/internal/api/packages/find.go

```diff
@@ -38,7 +39,20 @@ func findPackage(pattern *regexp.Regexp, w http.ResponseWriter, setDir func() (s
 		message.ErrorWebf(err, w, "Error getting directory")
 	}
 
+	// TODO: decide as a team how JSON errors should be formatted
+	type JSONError struct {
+		Error string `json:"error"`
+		Message string `json:"message"`
+	}
+
 	// Intentionally ignore errors
```

Comment:
This comment is throwing me off a little.

Seems like there is a bit of mixed intentions if we are ignoring the error from the `RecursiveFileList` call but then also returning an http error if the call didn't return any files. 

I would vote that we either:
1. Check the error and add it to an error return if it's not nil
2. Continue to not check the error but then also not return an error if empty (and expect the frontend to check/handle that)

Reply:
@jeff-mccoy You initially wrote the `// Intentionally ignore errors` so pinging you to pick your brain too

Reply:
Chose to handle error